portfolio/src/main/webapp/index.html

```diff
@@ -166,6 +166,29 @@ <h2>Connect with me</h2>
   <a class="nav-link" href="#">Return to Top</a>
 </div>
 <div id="comment-section">
+  <p>Have a feedback ? Please leave a comment
+  <button class="btn btn-primary" type="button" data-toggle="collapse" 
+      data-target="#comment-form" aria-expanded="false" aria-controls="comment-form">
+    Add Comment
+  </button>
+  <form action="/data" method="POST" class="collapse" id="comment-form">
```

Comment:
It's best to use lowercase for element names, attributes, etc. See https://google.github.io/styleguide/htmlcssguide.html#Capitalization

method='post'